community/consistency-check-legacy/src/main/java/org/neo4j/legacy/consistency/checking/full/FullCheck.java

```diff
@@ -161,7 +163,7 @@ static DiffRecordAccess recordAccess( StoreAccess store )
         T[] records = (T[]) Array.newInstance( type, (int) store.getHighId() );
         for ( int i = 0; i < records.length; i++ )
         {
-            records[i] = store.forceGetRecord( i );
+            records[i] = store.getRecord( i, store.newRecord(), FORCE );
```

Comment:
but that will not gonna create them more then it does currently, i'm proposing to have 2 methods: one that we already have - where we can pass object that we reuse, plus new one that will create new one internally for cases when we don't care about reuse